src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.includes(':'))
+    .reduce((obj, item) => {
+      const [property, ...value] = item.split(':');
+      const trimProperty = property.trim();
```

Comment:
The `trimProperty` variable is correctly defined and used; this is a good practice to ensure that the property names in the resulting object do not contain any leading or trailing whitespace.